.travis.yml

```diff
@@ -4,11 +4,15 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
+
+sudo:
+  false
 
 before_install:
   - phpenv config-rm xdebug.ini
   - composer self-update
-  - composer --prefer-source update
+  - composer --prefer-dist update
```

Comment:
This is for a purpose - there is no Github Auth key in this bundle, so all packages need to cloned. Check the builds, you get the `Could not authenticate against github.com. Now trying to download from source` error.